src/config.rs

```diff
@@ -108,10 +114,111 @@ pub fn clear_thread_local_config_option(key: &str) -> Result<()> {
     Ok(())
 }
 
+type CallbackType = dyn FnMut(CplErrType, i32, &str) + 'static + Send;
+type PinnedCallback = Pin<Box<Box<CallbackType>>>;
```

Comment:
You have to use the double-box in order to get a stable pointer and move something that boxes it around.
I could remove the `Pin` as it looks. The idea was to prevent it from moving around.